optimum/exporters/openvino/normalized_configs.py

```diff
@@ -0,0 +1,9 @@
+from optimum.utils import NormalizedTextConfig
+
+from .base import register_normalized_config
+
+
+@register_normalized_config("chatglm")
+class ChatGLM2NormalizedConfig(NormalizedTextConfig):
```

Comment:
same we could move it to optimum

https://github.com/huggingface/optimum/blob/e7bd60dd2c1e295263ba57a4e468a62ab5b179e8/optimum/utils/normalized_config.py#L80